packages/web-config-server/src/apiV1/dataBuilders/helpers/calculateOperationForAnalytics.js

```diff
@@ -92,6 +101,13 @@ const SINGLE_ANALYTIC_OPERATORS = ['CHECK_CONDITION', 'FORMAT', 'GROUP'];
 
 const ARITHMETIC_OPERATORS = ['DIVIDE', 'SUBTRACT'];
 
+export const getDataElementsFromCalculateOperationConfig = config =>
+  typeof config === 'string'
+    ? config
```

Comment:
Currently, `config` can't actually be a string when it is passed into `calculateOperationForAnalytics`. That case is handled earlier, in `getCalculatedValuesByCell`. Perhaps the `config === 'string'` check could be done in the matrix builder in order to make more sense?

Reply:
I totally agree!